crates/utils-sync/src/lib.rs

```diff
@@ -5,12 +5,14 @@ extern crate alloc;
 #[cfg(any(feature = "std", test))]
 extern crate std;
 
+pub mod once_lock;
 pub mod racy_lock;
 pub mod rw_lock;
 
 #[cfg(feature = "std")]
 pub use std::sync::LazyLock;
 
+pub use once_lock::OnceLockCompat;
```

Comment:
Similar to what we did with `LazyLock`, we could just call this one `OnceLock`

Reply:
I don't think that's wise, considering:
- it has an API much narrower than `OnceLock`,
- it has (due to the `Box`) a very different performance footprint in `no-std` environments.